src/Admin/AbstractAdmin.php

```diff
@@ -1762,11 +1762,13 @@ final public function hasAccess(string $action, ?object $object = null): bool
     }
 
     /**
+     * @param array<string, array<string, mixed>> $buttonList
+     *
      * @return array<string, array<string, mixed>>
      *
      * @phpstan-param T|null $object
      */
-    final public function getActionButtons(string $action, ?object $object = null): array
+    private function getInternalActionButtons(array $buttonList, string $action, ?object $object = null): array
```

Comment:
The first parameter of this function doesn't seem needed, it can be initialized inside the function to empty array.